e2e/test/context.go

```diff
@@ -14,6 +14,9 @@ import (
 	"go.uber.org/zap"
 )
 
+// Make it eaisr to manage namespaces created by the tests.
+const namespacePrefix = "e2e-"
```

Comment:
s/eaisr/easier/g

s/namespacePrefix/e2eNamespacePrefix/g

Second one is optional but it makes it clear.

Reply:
Maybe appNamespacesPrefix? Since we prefix only the app namespace. e2e is clear since we are in the context of e2e/test package.

Reply:
Replaced with appNamespacePrefix